### ISSUE TITLE
async (t) => { }    -  Expected parentheses around arrow function argument.

### ISSUE THREAD

**Issue by azhang** [feross/standard#416]
I'm getting `Expected parentheses around arrow function argument.` errors with my async arrow functions. Perhaps this is more of a eslint issue than a standardjs issue?

**Comment by feross** [comment]
There's already another discussion about this here: https://github.com/feross/standard/issues/414

**Comment by azhang** [comment]
Oh, actually I don't get that issue. I only get errors on async arrow functions. 

(t) => {} works
async (t) => {} errors

**Comment by feross** [comment]
Oh didn't realize this was only for `async` arrow functions.

**Comment by DiknT** [comment]
having the same issue with async arrow function with basic setup:
```
  "devDependencies": {
    "babel-eslint": "^4.1.8",
    "babel-preset-es2015-node5": "^1.1.2",
    "babel-preset-stage-0": "^6.3.13",
    "babel-register": "^6.4.3",
    "standard": "^6.0.4"
  }
```

**Comment by cesarandreu** [comment]
Having the same issue. This prevents us from upgrading to standard v6.

**Comment by azhang** [comment]
https://github.com/babel/babel-eslint/issues/252

**Comment by benjyhirsch** [comment]
This is a known issue with some of the rules in babel-eslint, hence https://github.com/babel/eslint-plugin-babel

From the description:

> An eslint plugin companion to babel-eslint. babel-eslint does a great job at adapting eslint for use with Babel, but it can't change the built in rules to support experimental features. eslint-plugin-babel re-implements problematic rules so they do not give false positives or negatives.

One of the rules is: 

> `babel/arrow-parens`: Handles async functions correctly

**Comment by cesarandreu** [comment]
This issue keeps us from upgrading, we're using async arrow functions a lot. Even if we add `eslint-plugin-babel`, according to #404 it seems v7 is gonna be removing "rules", so we'd have no way of replacing the eslint rule with the plugin rule?

Hmm, maybe making a `standard-babel` would make sense?

**Comment by feross** [comment]
@cesarandreu That's certainly one option, but I wonder if we shouldn't just disable/change this rule given the opposition to it in the other issue https://github.com/feross/standard/issues/414#issuecomment-185593164

**Comment by carlosvillu** [comment]
there is any workaround to use async + arrow functions ?!

Maybe a combination os standard version, eslint plugin, something like that ?!

**Comment by cesarandreu** [comment]
That'd also work :D. I've kinda grown to like it, just because it's consistent, but since it breaks with async arrow functions it's annoying :(.

**Comment by namuol** [comment]
TL;DR -- Here's a workaround that solves the problem:

```
npm install eslint-plugin-babel --save-dev
```

Then, in your eslintrc:

```
{
  // ...

  "plugins": [
    "babel"
  ],
  
  "rules": {
    // ...
    "arrow-parens": 0,
    "babel/arrow-parens": 2
  }
}
```

**Comment by feross** [comment]
This should be fixed now in `standard` version `7.0.0-beta.0` which removed the arrow function parens rule.
